.github/workflows/bot.yml

```diff
@@ -39,10 +39,12 @@ jobs:
   define-prs-matrix:
     name: Define PRs matrix
     # Skip this workflow if:
+    #   - the bot is not configured on this repo/fork
     #   - the bot is retriggering itself
     #   - the event is emitted by codecov
     #   - the event is a review on a pull request from a fork (see save-pr-number job below)
     if: |
+      vars.GH_BOT_LOGIN != '' &&
```

Comment:
nit: It's not immediately obvious that this is the line that skips the job running on a fork

Reply:
I don't think it's possible to add an inline comment in this case, which is why I made a comment above that explains condition by condition what it allows to filter.